scripts/zones/Crawlers_Nest_[S]/npcs/Tucker.lua

```diff
@@ -5,9 +5,11 @@
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    player:startEvent(14) -- doesn't want your trade dialog
```

Comment:
Need some confirmation here.  If you're not on the quest, does this always happen?  If it doesn't, then it can move to an else block in the quest (I'll help to verify)

Reply:
> Need some confirmation here. If you're not on the quest, does this always happen? If it doesn't, then it can move to an else block in the quest (I'll help to verify)

No, it does not. The Quest stuff overrides it.

Reply:
Right, but what happens if you trade to him when you're _not_ on the quest? This is the behavior I'm asking about here